querydsl-sql/src/test/java/com/querydsl/sql/AbstractSQLTemplatesTest.java

```diff
@@ -70,19 +70,19 @@ public void union() {
 
         if (templates.getDummyTable() == null) {
             assertEquals(
-                    "(select 1 as col1)\n" +
```

Comment:
Is it a good idea to reuse the wrapping parameter when asserting the SQL?  Means the parentheses is built in asserted SQL string when the wrapping is true?